### DISCUSSION TITLE
Issue with LCDM distance

### DISCUSSION THREAD

**Discussion by RichardRavenhall** [a-cosmology-group/acg#42] [Astrophysics and Physics]
<html xmlns:o="urn:schemas-microsoft-com:office:office"
xmlns:w="urn:schemas-microsoft-com:office:word"
xmlns:m="http://schemas.microsoft.com/office/2004/12/omml"
xmlns="http://www.w3.org/TR/REC-html40">

<head>

<meta name=ProgId content=Word.Document>
<meta name=Generator content="Microsoft Word 15">
<meta name=Originator content="Microsoft Word 15">
<link rel=File-List
href="file:///C:/Users/Owner/AppData/Local/Temp/msohtmlclip1/01/clip_filelist.xml">
<link rel=themeData
href="file:///C:/Users/Owner/AppData/Local/Temp/msohtmlclip1/01/clip_themedata.thmx">
<link rel=colorSchemeMapping
href="file:///C:/Users/Owner/AppData/Local/Temp/msohtmlclip1/01/clip_colorschememapping.xml">

<style>

</style>

</head>

<body lang=EN-US style='tab-interval:.5in;word-wrap:break-word'>

<p style='margin:0in;background:white'><b><span style='font-family:"Calibri",sans-serif;
color:black'>Webb observes a luminosity distance in conflict with LCDM distance
prediction.</span></b><b><span style='font-family:"Calibri",sans-serif'><o:p></o:p></span></b></p>

<p style='margin:0in;background:white'><span style='font-family:"Calibri",sans-serif'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal style='margin-bottom:0in;line-height:normal;text-autospace:
none'><span style='font-size:12.0pt'>Per the 6/6/23 paper “JADES. Possible
Population III signatures at z=10.6 in the halo of GN-z11”, R. Maiolino et al,
there is a high probability that LCDM predicted galaxy distances are
inconsistent with Webb findings.&nbsp; This is due to recent Webb observations
of an exceptionally high luminosity galaxy in close alignment (and at z =
10.603) to the super bright galaxy GN-z11 (at z = 10.957).&nbsp; Webb finds an
~ 20 Gly luminosity distance that is approximately 35% to 40% lower than the 32
Gly predicted by LCDM.&nbsp; <o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:0in;line-height:normal;text-autospace:
none'><span style='font-size:12.0pt'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal style='margin-bottom:0in;line-height:normal;text-autospace:
none'><span style='font-size:12.0pt'>The following is a copy of the paper’s
introducing the information:   https://arxiv.org/pdf/2306.00953.pdf <o:p></o:p></span></p>

“Here we present NIRSpec-IFU and NIRSpec-MSA observations of the   region around GN-z11, an exceptionally luminous galaxy at z=10.6, which   reveal a >5σ detection of a feature consistent with being HeII λ1640   emission at the redshift of GN-z11. The very high equivalent width of the   putative HeII emission in this clump (170 A), and the lack of metal lines,   can be explained in terms of photoionisation by PopIII stars, while   photoionisation by PopII stars is inconsistent with the data. It would also   indicate that the putative PopIII stars likely have a top-heavy initial mass   function (IMF), with an upper cutoff reaching at least 500 M⊙. The PopIII bolometric   luminosity inferred from the HeII line would be ∼2×1010L⊙, which (with a top-heavy IMF)   would imply a total stellar mass formed in the burst of ∼6×105M⊙.”  Cited in: Astrophysics   of Galaxies (astro-ph.GA); Cosmology and Nongalactic Astrophysics   (astro-ph.CO).
--

<p style='margin:0in;background:white'><span style='font-family:"Calibri",sans-serif'><o:p>&nbsp;</o:p></span></p>

<p style='margin:0in;background:white'><span style='font-family:"Calibri",sans-serif;
color:black'>Mainstream cosmology assumes the correctness of LCDM distances by
verifying them via the “Cosmology Distance Ladder”.&nbsp; There are many who
challenge the ladder’s viability, though this measurement framework has yet to
be proven “correct” or “incorrect”.&nbsp; Yet, the extremely improved accuracy
of Webb’s data may provide new physics.&nbsp; Thus, the difference between
these two approaches could have major significance.</span><span
style='font-family:"Calibri",sans-serif'><o:p></o:p></span></p>

</body>

</html>

**Comment by RedshiftDrift** [comment]
@RichardRavenhall you write
> exceptionally high luminosity galaxy in close alignment (and at z = 10.603) to the super bright galaxy GN-z11 (at z = 10.957)

However, the paper only mentions one galaxy, GN-z11, at redshift z=10.603.  (The foreground galaxy at $z\approx 2$ does not contribute to the HeII spectrum.)  On p. 4:
> The 2D spectrum shows a tentative signal at 1.905 μm, which consistent with the expected location of HeIIλ1640 at the redshift of GN-z11 (z=10.603), peaking at about 0.5′′ from GN-z11.

The "clump" is not another galaxy but "HeII detection in the halo of GN-z11", as explicitly written in $\S3.1$.  The paper does not report a distance, but the spectrum of the ionized halo of GN-z10.

**P.S.** For papers on arXiv it is desirable to give the link to the abstract [https://arxiv.org/abs/2306.00953](https://arxiv.org/abs/2306.00953), never to the pdf version of the article.

**Comment by khuramonline** [comment]
Circular Logic vs circular logic.

"There is difference between LCDM distance and Luminosity based distance" "noted" by JWST.

As if this "Luminosity based distance" does not include LCDM based formulas.

**Reply by RichardRavenhall** [reply]
Luminosity based distance in this case has significance because it doesn't include LCDM formulas.

**Reply by RichardRavenhall** [reply]
I back off my above reply.  Do we have anybody that is knowledgeable about the subject?

**Reply by khuramonline** [reply]
> I back off my above reply. Do we have anybody that is knowledgeable about the subject?

Luckily I just recently had a conversation about exact same topic i.e. "Luminosity based distance of GNz-11" with Chat-GPT. Conversation date was 24-05-2023.

At first GPT did surprise me by stating that Luminosity distances are also based on Cosmological Model used.

![image](https://github.com/a-cosmology-group/acg/assets/132253587/630d944e-de06-4d87-b9b4-6619a99102fc)

The conversation proceeded. After few other questions, then I asked following question about pure D_L (Luminosity distance) of GNz-11:

![image](https://github.com/a-cosmology-group/acg/assets/132253587/375acab9-cfc0-43e4-a2a9-44b6bf7c8327)

Here I was not satisfied by the answer. The conversation proceeded. After few other questions, then I asked following question:

![image](https://github.com/a-cosmology-group/acg/assets/132253587/12da29ab-fe34-485e-a2d6-f757fef26a55)

The conversation proceeded. After another question, I asked following question:

![image](https://github.com/a-cosmology-group/acg/assets/132253587/0d6df620-4aa4-4730-b147-b5e2fcd95a82)

In further conversation, the GPT provided the value of 282 ... That was ambiguous value because GPT avoided to confirm or exactly tell that it is distance in billion light years.

Anyways, my original stance stands. 

Another thing I want to say is that Luminosity is pure empirical thing. Luminosity based OFFICIAL distance cannot be in terms of Comoving distance of 20 BLY or 32 BLYs.

What is Empirical is ONLY the light travel distance. Luminosity of only the light travel distance can be noted.

If they have told us luminosity distance in terms of comoving distance then obviously they have employed CIRCULAR logic. They DO HAVE CONTAMINATED the luminosity distances with LCDM based formulas.

**Comment by RichardRavenhall** [comment]
Things come down to different interpretations and I offer what could be a different perspective and possible new information.  The clump is assumed to be a small galaxy at z = 10.603 near Gn-z11 which we know has a z = 10.957.

The paper describes work that is looking for PopIII stars, not distances.  It presents Webb observations of the region around GN-z11 and finds an exceptionally luminous galaxy at z=10.6.  It also indicates an associated luminosity distance of ∼2×1010 L⊙ or ~20 Gly.  

There can only be one correct distance.  Is it ~20 Gly as inferred from Webb observations, 32 Gly as stipulated by LCDM, or completely different?  

Note that the ~20E9 number only appears in the paper’s abstract and is not addressed again.  Could this have significance?

**Reply by RedshiftDrift** [reply]
See the updated value for the redshift of GN-z11, z=10.60, see [arXiv:2302.07234](https://arxiv.org/abs/2302.07234).

> Gn-z11 which we know has a z = ~10.957~

No, the ***incorrect*** value ~10.957~ reported in an article published in 2021[^1] is based on a "*probable* detection of three ultraviolet emission lines" which turned out to be wrong.

The galaxy GN-z11 and the halo (clump) are all part of the same structure at z = 10.603, there is nothing at z=~10.957~.

[^1]: Incorrect value published in [Nature Astronomy volume 5, pages 256–261 (2021)](https://www.nature.com/articles/s41550-020-01275-y)

**Comment by RichardRavenhall** [comment]
Wow - so GN-z11's z is not 10.957.  This could mean that all galaxy z's are not as precisely known as we believed.  Does mainstream cosmology agree with this finding?

**Reply by RedshiftDrift** [reply]
This is common practice, the papers [arXiv:2302.07234](https://arxiv.org/abs/2302.07234) and [arXiv:2306.00953](https://arxiv.org/abs/2306.00953) report GN-z11 at z=10.60 but there is still a small risk of error.

These numbers are never set in stone.  The 2021 paper mentioned a "***probable*** detection" to indicate that the spectroscopic lines may not be identified correctly.  Looking at the spectrum, the signal-to-noise was not that great, so it's not surprising they picked the wrong lines.  And when the redshift is only *photometric* (like most of JWST galaxies), the galaxies are said to be high-redshift ***candidates***!  The published values of *z* are accurate to 10% unless there is an interpretation error, in which case the value of *z* can be off by a factor of three!
Every astronomer accepts that.

>galaxy z's are not as precisely known as we believed.

One has to read between the lines, every measurement is at risk of an incorrect interpretation.

**Reply by RichardRavenhall** [reply]
I appreciate both of your replies but suspect that none of us has changed anybody else's original views.  Hopefully the over 20 billion spent on Webb will help us find a better answer.
